### ISSUE TITLE
Can't create any event due to empty "saving to" combo box

### ISSUE THREAD

**Issue by sdeleuze** [etesync/etesync-web#43]
On a newly created account, I can't create any event in the calendar on https://client.etesync.com, potentially because the "saving to" combo box is empty and I can't find how to add a new entry via the web interface.

Is it a bug or did I miss something? I have no Android device and I would like to be sure it is usable without the experimental hosted DAV adapter for basic use case.

**Comment by tasn** [comment]
It should work without an Android device, and you definitely don't need the hosted DAV adapter for normal operation...

Could you try logging out from the web client and then logging in again? Does that fix anything?
Which browser are you using?
What do you see in https://client.etesync.com/journals ? Is there anything under Address books? Anything under calendars? (Should list journals there, e.g. "Default")

I just tested it here and it works as expected.

Could you please email support so I can help you look into it?

**Comment by sdeleuze** [comment]
I am using Firefox 66 on Ubuntu 19.04.
Login/logout does not fix the issue.

I see this error in the devtools:
```
NetworkError: Failed to load worker script at https://storage.googleapis.com/workbox-cdn/releases/3.6.3/workbox-sw.js (nsresult = 0x805e0006) service-worker.js:14
Error during service worker registration: TypeError: "ServiceWorker script at https://client.etesync.com/service-worker.js for scope https://client.etesync.com/ encountered an error during installation." main.6aee4ee3.chunk.js:1:103938
    li https://client.etesync.com/static/js/main.6aee4ee3.chunk.js:1
```

In journals I just see:
- Address Books
- Calendars
- Tasks

No default calendar.

I will send a mail to the support.

**Comment by tasn** [comment]
The errors aren't related. It's because for some reason the create-react-app is now trying to fetch service worker libraries from Google and we use very strict CSP rules to block requests to third parties.

**Comment by tasn** [comment]
This was indeed an issue, fixed with a557ec1fd26b9422c6548872e9c144362a297fb6.

Thanks for reporting!
